.github/workflows/ci.yml

```diff
@@ -103,8 +103,8 @@ jobs:
           DD_API_KEY: ${{ secrets.DD_API_KEY_CI_VISIBILITY }}
           DD_APP_KEY: ${{ secrets.DD_APP_KEY_CI_VISIBILITY }}
           DD_SERVICE: datadog-ci-e2e-tests-junit
-      - name: Run junit upload test (metrics)
-        run: yarn datadog-ci junit upload --service=datadog-ci-e2e-tests-junit artifacts/e2e/junit-reports --metrics testmeasure2:60 --report-metrics sessionmeasure2:80
+      - name: Run junit upload test (metrics), passing glob as literal string
+        run: yarn datadog-ci junit upload --service=datadog-ci-e2e-tests-junit '**/junit-reports/**' --metrics testmeasure2:60 --report-metrics sessionmeasure2:80
```

Comment:
by passing quotation marks the pattern is not expanded and is managed by the CLI. This makes for a more complete test and also helps testing the usage of https://github.com/DataDog/junit-upload-github-action/